kit/src/components/message/mod.rs

```diff
@@ -408,6 +408,14 @@ pub fn ChatText(cx: Scope<ChatMessageProps>) -> Element {
 }
 
 pub fn format_text(text: &str, should_markdown: bool, emojis: bool) -> String {
+    // warning: this will probably break markdown regarding block quotes. still seems like an improvement.
+    let mut safe_text = String::from(text);
+    safe_text = safe_text.replace("&", "&amp;");
+    safe_text = safe_text.replace("<", "&lt;");
+    safe_text = safe_text.replace(">", "&gt;");
+    safe_text = safe_text.replace("\"", "&quot;");
+    safe_text = safe_text.replace("'", "&#x27;");
```

Comment:
You could chain it doing the following

```suggestion
  let safe_text = text
        .replace("&", "&amp;")
        .replace("<", "&lt;")
        .replace(">", "&gt;")
        .replace("\"", "&quot;")
        .replace("'", "&#x27;");
```